Configuration/PyReleaseValidation/scripts/runTheMatrix.py

```diff
@@ -206,6 +212,7 @@ def runSelected(opt):
                       action='store_true')
     
     opt,args = parser.parse_args()
+    print opt.nThreads
```

Comment:
Yes, I pointed this out to Giulio, and he said he’d fix them on integration since we don’t want the bot to fire just for this change…

-Liz
On Feb 24, 2015, at 9:43 AM, Eric Vaandering <notifications@github.com<mailto:notifications@github.com>> wrote:


In Configuration/PyReleaseValidation/scripts/runTheMatrix.py<https://github.com/cms-sw/cmssw/pull/7754#discussion_r25261695>:

> @@ -206,6 +212,7 @@ def runSelected(opt):
>                        action='store_true')
>
>      opt,args = parser.parse_args()
> +    print opt.nThreads


Same here

—
Reply to this email directly or view it on GitHub<https://github.com/cms-sw/cmssw/pull/7754/files#r25261695>.